packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -290,7 +290,7 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     actions.comments.getSandboxComments(sandbox.id);
   }
 
-  if (sandbox.originalGit) {
+  if (sandbox.originalGit && sandbox.owned) {
```

Comment:
Oh! Can we change this one to `hasPermission(sandbox.authorization, 'owner')`? I slowly want us to start using this new permission system, this also dynamically updates with live 😄

Reply:
Fixed 👍